lib/elixir/lib/kernel/typespec.ex

```diff
@@ -477,26 +477,26 @@ defmodule Kernel.Typespec do
     typespec({:record, meta, [atom, []]}, vars, caller)
   end
 
-  defp typespec({:record, meta, [atom, fields]}, vars, caller) do
+  defp typespec({:record, meta, [tag, field_specs]}, vars, caller) do
```

Comment:
I tried to match the naming with `Record.defrecord`

Reply:
:+1: